### ISSUE TITLE
(Ctrl + Shift + Alt + Numpad) Win10 not working

### ISSUE THREAD

**Issue by thewaliii** [craigmdennis/adobe-xd-nudge-and-resize#17]
First of all, this is an awesome plugin
At the moment the shortcuts are:
"Ctrl+ Alt + Numpad"  for shrink/extend
"Ctrl+ Shift + Alt + Arrows" for large shrink/extend
Which is not consistent.
"Ctrl+ Shift + Alt + Numpad" doesn't work in windows 10

**Comment by craigmdennis** [comment]
I don't have a windows machine (or keyboard with a number pad) to test on.
If I recall, "Ctrl+ Alt + Arrows" is a reserved keyboard shortcut that I can't use.

I'll look at what the options are regarding replacing the keyboard shortcut on Windows.

**Comment by Nickyg001** [comment]
Yes, I still haven't understood in windows 10 how to move with keyboard commands with this plug-in ...

Oh well, the strangest thing I don't understand is why it's not Abobe itself that implements the 8px shift function or can be customized at will!
